tests/test_client_functional.py

```diff
@@ -1439,7 +1439,7 @@ async def handler(request):
                                         'application/octet-stream',
                                         'text/x-python']
         assert request.headers['content-disposition'] == (
-            "inline; filename=\"conftest.py\"; filename*=utf-8''conftest.py")
```

Comment:
You remove tests for `filename*` but I'd prefer to have them with actually correct examples with Unicode filenames instead.

Reply:
`test_helpers.py` has tests with unicode filenames. They do not carry the `filename*` because RFC7578 tells not to use it here.